styles/components/alert.scss

```diff
@@ -1,3 +1,40 @@
 .alert {
+	padding: 20px 40px;
+	margin: 10px;
+	position: relative;
+	color: $brand-white;
 
+	.dismiss-btn {
```

Comment:
I don't think this belongs to `alert`. You may remove this ( as well as the example ) .